sos/plugins/pacemaker.py

```diff
@@ -26,6 +26,11 @@ def setup(self):
         self.add_copy_spec([
             "/var/lib/pacemaker/cib/cib.xml"
         ])
+
+        self.limit = self.get_option("log_size")
```

Comment:
Why define an instance variable for something that is used exactly once?

Just make the call in-line at the site of `self.add_copy_spec_limit()`.